frontend/web/project/api.js

```diff
@@ -44,6 +44,10 @@ global.API = {
       mixpanel.alias(id)
     }
 
+    if (typeof DYNATRACE_URL !== 'undefined' && !!DYNATRACE_URL) {
```

Comment:
I'd prefer to see this code not written out multiple times, can we use the `enableDynatrace` var from a central location somehow? Or just redefine it here so we're at least only doing the check once.

Reply:
Done